### ISSUE TITLE
Bambu lab cloud no response !!!!

### ISSUE THREAD

**Issue by Stefbull** [bambulab/BambuStudio#4791]
### Bambu Studio Version

1.9.5.51

### Where is the application from?

Bambu Lab Official website

### OS version

win11 64bit

### Additional system information

i9-rtx4070-32gbram

### Printer

X1c

### How to reproduce

![Capture d'écran 2024-09-10 114039](https://github.com/user-attachments/assets/0e4dfb72-cc0e-4d78-8aa3-e9d0b1b5a3d9)

### Actual results

?

### Expected results

?

### Project file & Debug log uploads

?

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by fluppir** [comment]
Yes me too - looks like whatever cloud maintenance they had, has failed some.

**Comment by ttt888zzz** [comment]
Yesterday, the official U.S. server migrate moved from California to Oregon, and a system anomaly occurred between 17:08 and 20:36, causing access problems.Live announcements were posted on [status.bambulab.com](http://status.bambulab.com/)。
Please try again. We are very sorry for the inconvenience。
